docs/output.md

```diff
@@ -124,10 +124,36 @@ The resulting FASTQ files will only be present in your results directory if you
   - `*.{bai,csi}`: Index file corresponding to a BAM file which is for faster downstream steps (e.g. SAMtools).
   - `*.flagstat`: Statistics of aligned reads from SAMtools `flagstat`.
 
-  </details>bam,bai,csi
+  </details>
 
   [BWA](https://bio-bwa.sourceforge.net/bwa.shtml) is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome. It consists of three algorithms: BWA-backtrack (a.k.a `bwa aln`), BWA-SW and BWA-MEM. The first algorithm is designed for Illumina sequence reads up to 100bp, while the rest two for longer sequences ranged from 70bp to 1Mbp. BWA-MEM and BWA-SW share similar features such as long-read support and split alignment, but BWA-MEM, which is the latest, is generally recommended for high-quality queries as it is faster and more accurate. BWA-MEM also has better performance than BWA-backtrack for 70-100bp Illumina reads.
 
+### BAM Filtering
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bam_filtering/`
+
+  - `*length.filtered.bam`: BAM file containing mapped and unmapped reads with length filtered reads.
+  - `*.filtered.bam`: BAM file containing mapped quality filtered reads (and optionally length filtering and unmapped reads, if specified by the user with the corresponding parameters).
```

Comment:
> (and optionally length filtering and unmapped reads, if specified by the user with the corresponding parameters).
Isn't that the same as the one above?

Reply:
No, this one may have JUST quality filered reads, or BOTH quality and length filtered; they are additive